ship.config.js

```diff
@@ -15,16 +14,6 @@ module.exports = {
   publishCommand({ tag }) {
     return `yarn publish --access public --tag ${tag}`;
   },
```

Comment:
Was it previously done by `lerna`?

Reply:
Yes, here's what it does ([source](https://github.com/lerna/lerna/blob/main/commands/version/README.md)):

> 1. Identifies packages that have been updated since the previous tagged release.
> 2. Prompts for a new version.
> 3. Modifies package metadata to reflect new release, running appropriate [lifecycle scripts](https://github.com/lerna/lerna/blob/main/commands/version/README.md#lifecycle-scripts) in root and per-package.
> 4. Commits those changes and tags the commit.
> 5. Pushes to the git remote.

This should be handled by Ship.js, but we'll see how it goes in the next release.